pkg/azureutils/azure_disk_utils.go

```diff
@@ -115,12 +116,16 @@ const (
 	// to prevent the pod deletion until clean up is completed
 	ControllerFinalizer              = "disk.csi.azure.com/azuredisk-finalizer"
 	VolumeAttachmentExistsAnnotation = "disk.csi.azure.com/volume-attachment"
+	CleanUpAnnotation                = "disk.csi.azure.com/clean-up"
 	VolumeDetachRequestAnnotation    = "disk.csi.azure.com/volume-detach-request"
 	VolumeDeleteRequestAnnotation    = "disk.csi.azure.com/volume-delete-request"
 	NodeNameLabel                    = "node-name"
 	VolumeNameLabel                  = "volume-name"
 	RoleLabel                        = "requested-role"
```

Comment:
Belated observation, but I wonder if we should prefix the driver-specific labels with `disk.csi.azure.com`?

Reply:
I think we should. I will update that part as well in this PR.